ui/src/components/AlertDrawer/index.tsx

```diff
@@ -13,7 +13,12 @@ export default function AlertDrawer({
   ...props
 }: AlertRuleDrawerProps) {
   return (
-    <Drawer closable={false} onClose={onClose} {...props}>
+    <Drawer
```

Comment:
Add `maskClosable={false}` here to avoid accident closing.

Reply:
Done.